app/models/lesson.rb

```diff
@@ -37,4 +37,12 @@ def delete_associated_asl_files
   def duration_str
     Duration.duration_str(duration)
   end
+
+  def duration_to_int(duration_param)
```

Comment:
What if we better handled the ":" in the duration_str_to_int method, then we'd be able to remove this method all together and then be able to call @lesson.duration = Duration.duration_str_to_int(lesson_params[:duration]).  That would be a bit simpler, right?